spec/models/book_spec.rb

```diff
@@ -200,4 +200,121 @@
       it { expect(Chapter.count).to eq 1 }
     end
   end
+
+  describe 'enabled chapters' do
```

Comment:
I don't know if it's intentional, but a lookahead of 0 behaves like fully display. I think it's the expected behavior but we should either add a test for it or validate that only positive values are given.

Reply:
Nice catch.

Yeah, it is an expected behaviour, because that would mean otherwise that you can't see even the content you are working in. 

I'll add a test and a validation and merge, does it make sense?

Reply:
Yesss :smile: